lib/wallet/api/http/Cardano/Wallet/Api.hs

```diff
@@ -604,6 +604,7 @@ type ListTransactions n = "wallets"
     :> QueryParam "end" Iso8601Time
     :> QueryParam "order" (ApiT SortOrder)
     :> QueryParam "max_count" ApiLimit
+    :> QueryParam "addressId" (ApiAddressIdT n)
```

Comment:
```suggestion
    :> QueryParam "address" (ApiAddressIdT n)
```

?

Reply:
I agree here, but complied with what is in `InspectAddress` and `PutByronAddress`. Nevertheless, address seems less confusing than addressId. We might want to change the legacy "addressId" in the future.